src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -63,6 +65,9 @@ public void send(PaymentCommand cmd) {
 
     private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
         try {
+            LOG.info("Sending message to payments queue! ID: {}, Label: {}",
```

Comment:
How about logging this after the message was sent (or having an additional log entry there)?

Reply:
Why the exclamation mark?

Reply:
I think we can remove these logs after the tests

Reply:
The fact that we have sent a message is well worth logging.